benefit/data/sheets/CaseEventToFields/CaseEventToFields-dwpUploadResponse.json

```diff
@@ -175,7 +175,7 @@
   },
   {
     "CaseEventID": "dwpUploadResponse",
-    "CaseFieldID": "dwpState",
+    "CaseFieldID": "dynamicDwpState",
```

Comment:
Is there any reason to keep the previous field id name? Does this affect the dwpState field in other areas outside the ticket scope?

Reply:
dwpState is the state on the case, this needs to stay the same. this dynamic list will set the "dwpState" at the end of the event, its just a temporary field for the upload response event